instagram/src/styles/ProfileEditPage/DetailsForm_style.jsx

```diff
@@ -0,0 +1,24 @@
+import styled from "@emotion/styled";
+
+export const ItemForm = styled.form`
+  width: 700px;
+  height: 597px;
+  margin: 16px 0;
+`;
+
+export const SubmitBtn = styled.div`
```

Comment:
SubmitBtn과 div는 안맞음 button로 교체
```suggestion
export const SubmitBtn = styled.button`
```